ravenframework/SupervisedLearning/ROMCollection.py

```diff
@@ -903,6 +903,7 @@ def evaluate(self, edict):
         result, weights = self._createNDEvaluation(edict)
       clusterStartIndex = 0 # what index does this cluster start on in the truncated signal?
       globalLocalPicker = []
+      nRoms = len(self._roms)
```

Comment:
does this variable get used later?

Reply:
I don't see it showing up in the rest of the method, so I think we can remove it. I'm sure I had something planned for it that never came to fruition.